manager/manager.go

```diff
@@ -142,12 +144,15 @@ func New(config *Config) (*Manager, error) {
 		tcpAddr = net.JoinHostPort("0.0.0.0", tcpAddrPort)
 	}
 
-	err := os.MkdirAll(filepath.Dir(config.ProtoAddr["unix"]), 0700)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to create socket directory")
+	// don't create a socket directory if we're on windows. we used named pipe
+	if runtime.GOOS != "windows" {
+		err := os.MkdirAll(filepath.Dir(config.ProtoAddr["unix"]), 0700)
```

Comment:
no it definitely is supposed to be `GOOS != "windows"`. if the OS is not windows we create a directory. we don't need to make a directory for windows named pipes; they exist in their own, like, space.